### DISCUSSION TITLE
Using Sauron purely for typed HTML generation server side

### DISCUSSION THREAD

**Discussion by yatesco** [ivanceras/sauron#52] [Q&A] [answered]
Hi - looking at the progressive rendering section, is there any reason why I couldn't just use Sauron as a pure server side technology to generate HTML in a type safe manner?

I'm thinking not, but I thought I'd check!

**Comment by ivanceras** [comment, marked as answer]
Yes, you can omit the code in the index page where it links the javascript code.

Just remove this code in `index.html` or the page template code in the server side code.
```html
<script type=module>
        import init from './pkg/counter.js';
        await init().catch(console.error);
</script>
```
